CHANGES

```diff
@@ -50,6 +50,7 @@
 - Android & iOS: Use empty Audio implementations on Android and iOS when audio is disabled
 - Adds status detection for an httpRequest
 - Fix ANGLE GLES renderer on dekstop (#7274)
+- Architecture support: Support for Linux RISC-V has been added. The gdx-xxx-natives.jar files now also contain native libraries for this architecture.
```

Comment:
I would rather mention the gradle artifact here. That the `natives-desktop` now contains RISCV binaries.